internal/operator-controller/rukpak/preflights/crdupgradesafety/crdupgradesafety.go

```diff
@@ -171,7 +172,11 @@ func sameVersionErrors(results *runner.Results) []error {
 		for property, comparisonResults := range propertyResults {
 			for _, result := range comparisonResults {
 				for _, err := range result.Errors {
-					errs = append(errs, fmt.Errorf("%s: %s: %s: %s", version, property, result.Name, err))
+					msg := err
+					if result.Name == "unhandled" {
+						msg = "unhandled changes found"
+					}
+					errs = append(errs, fmt.Errorf("%s: %s: %s: %s", version, property, result.Name, msg))
```

Comment:
it feels like we might be occluding the error. Is it possible to format it it in a better way? Then maybe at the end we limit the number of errors we print out?

Reply:
I am not sure if I could understand your suggestion here. 
Could you please clarify?

Reply:
I could improve by extracting info from the JSON. 
See wdyt now

Reply:
HI @anik120 

We MUST do this change.
Any further improvements than is in the lib